### ISSUE TITLE
Bug in LiveSplit TimeSpan Parsing / Export

### ISSUE THREAD

**Issue by CryZe** [glacials/splits-io#268]
Turns out that .NET encodes times in a really weird way when times get larger than 1 day. 128 hours get encoded like this in the LSS file:

```xml
<BestSegmentTime>
    <RealTime>5.08:00:00</RealTime>
</BestSegmentTime>
```

We had the same issue in LiveSplit One, so I figured splits i/o might have that issue too, and indeed it does. Here's the splits file with a time of 128 hours, that fails to get parsed properly: https://splits.io/12mv

**Comment by glacials** [comment]
cool thanks, agreed weird. i'll update the parser

**Comment by CryZe** [comment]
And exporting too

**Comment by glacials** [comment]
ah yeah

**Comment by BatedUrGonnaDie** [comment]
Closed with #312
